hail/python/hail/genetics/reference_genome.py

```diff
@@ -226,8 +226,8 @@ def contig_length(self, contig):
         else:
             raise KeyError("Contig `{}' is not in reference genome.".format(contig))
 
-    @typecheck_method(contig=str)
-    def _contig_global_position(self, contig):
+    @property
+    def _global_positions_dict(self):
```

Comment:
Hmm. Any reason to not make this part of the public API (make it not underscore)? If we're referencing it in the plotting functionality, then it must be generally useful, yeah?

Reply:
Forgot this was in there, modified quickly as part of a "genomic" scale experiment. Will clean up and make it not `_`.